raiden/messages.py

```diff
@@ -1075,6 +1106,7 @@ def _data_to_sign(self) -> bytes:
             nonce=self.balance_proof.nonce,
             monitoring_service_contract_address=self.monitoring_service_contract_address,
         )
+        # TODO: should we add cmdid?
```

Comment:
For consistency it could be interesting to have it an additionally hash to the `MonitoringRequest`.

However it doesn't seem necessary ATM.